Dockerfile

```diff
@@ -25,6 +25,12 @@ ARG BUILD_DATE="NULL"
 ARG VERSION="NULL"
 ARG VCS_REF="NULL"
 
+# Install AutonomousStuff Delphi SRR2 Driver Package
+RUN sudo apt update && sudo apt install apt-transport-https && \
```

Comment:
Think you might be able to do without this `apt-transport-https` install, if I recall that's already installed in the base image. Saves you from doing 2 apt updates at least.